### DISCUSSION TITLE
Question about pwpush-ephemeral:1.29.3 and PWPUSH_MASTER_KEY variable

### DISCUSSION THREAD

**Discussion by CBB3810** [pglombardo/PasswordPusher#2406] [Q&A] [answered]
I just tried encryption key settings in version 1.29.3 and it doesn't work. Months ago, I saved this version as a custom docker image locally because when using externally mounted files, anything above version 1.29.3 was broken and wouldn't start up. See previous thread from me (#1363) 

Should I pull your latest default YAML and diff and sync with my old yaml where settings need changed and try your latest image? What version did the encryption variable become in use?

**Comment by CBB3810** [comment, marked as answer]
Peter, I was able to solve this. I did grab the latest yaml posted on github and did all the necessary config customization's I had set. I tried launching with latest and using a "-e PWPUSH_MASTER_KEY=xxxxxxxxx" environment variable on the launch and I'm back to using the latest image with custom encryption key. This one can be marked as answered. Thank you.

**Reply by CBB3810** [reply]
My launch command looks like:
docker run -d -e PWP__THEME=slate -e PWP_PRECOMPILE=true -e PWPUSH_MASTER_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx -p 0.0.0.0:5100:5100 --mount type=bind,source=/opt/dockerdata/sql_lite/db.sqlite3,target=/opt/PasswordPusher/db/db.sqlite3 -v /opt/dockerdata/logos:/opt/PasswordPusher/public/logos --mount type=bind,source=/opt/dockerdata/yaml/settings.yml,target=/opt/PasswordPusher/config/settings.yml pglombardo/pwpush-ephemeral:latest

I'll probably save this as a custom docker image and use it until I need to change again. It's easier to keep production stable this way.

**Reply by pglombardo** [reply]
Eventually all those YAML configs - or at least the large majority will move inside the application.  Great to hear it's working for you now.
